scripts/build-docs.js

```diff
@@ -31,7 +31,7 @@ if (isPullRequest) {
   const newPkg = { ...pkg, version: `999.999.999-pr.${process.env.REVIEW_ID}` };
   pipe(JSON.stringify(newPkg, null, 2)).to("package.json");
   shell.exec("yarn build --playground");
-  pipe(JSON.stringify(pkg, null, 2) + "\n").to("package.json"); // restore
+  pipe(`${JSON.stringify(pkg, null, 2)}\n`).to("package.json"); // restore
```

Comment:
Seriously, this rule is too aggressive.

Reply:
@fisker Seriously, what's wrong with `+ '\n'`?

Reply:
Nothing wrong, but also nothing wrong use template literal, think this way, this rule is opinionated,  just live with it.

Sometimes like Prettier self, in many places, user don't have a choice, have to accept what ever Prettier choose.

Reply:
In the case of Prettier the user gets a lot of benifits in return. I don't see enough benifits in this case.

Reply:
All those opinionated decisions in Prettier aren't needed just for the sake of themselves. They're needed to make Prettier work. Prettier won't work at all without those decisions. They're a neccessity. Nothing like these ESLint rules that are just linting for the sake of linting.

Reply:
@thorn0 What about this https://github.com/prettier/prettier/pull/10175#issuecomment-769771050?

Reply:
You said yourself, many changes are okay, shouldn't that enough?

Reply:
No, it's not enough. It means the rule isn't good enough. It needs an improvement, a less aggressive mode that won't uglify simple binary expressions like `something + '\n'`

Reply:
@fisker I'd like to emphasize again if my point wasn't clear that you can't compare things like this to Prettier. They're incomparable.

The benefits you get in return when you accept Prettier's opinionated decisions: Prettier itself with all its benefits.
The benefits you get in return when you accept ugliness that comes from enforcing `prefer-templates`: some expressions now look better. Ehm... ⚖

Reply:
Checkout this PR, revert lines you don't like, turn off the rule, commit. Tools suppose to help us, not force us. I offen turn on some rules I usually don't use, just to check possible problems and see if improvements.

What do you say @lipis?